hardware.inc

```diff
@@ -33,19 +34,21 @@ HARDWARE_INC SET 1
 rev_Check_hardware_inc : MACRO
 ;NOTE: REVISION NUMBER CHANGES MUST BE ADDED
 ;TO SECOND PARAMETER IN FOLLOWING LINE.
-    IF  \1 > 2.9 ;PUT REVISION NUMBER HERE
+    IF  \1 > 3.0 ;PUT REVISION NUMBER HERE

```

Comment:
Why not 2.10?

Reply:
It looks odd when 2.1 already exists, and for consistency: 1.9 went to 2.0. Also, the change is big enough to warrant a more significant rev.